exercises/concept/chessboard/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+# Instructions
+
+As a chess enthusiast, you would like to write your own version of the game. Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
```

Comment:
Single Sentence Per Line as per [markdown specification] for Exercism.

[markdown specification]: https://exercism.org/docs/building/markdown/markdown

```suggestion
As a chess enthusiast, you would like to write your own version of the game.
Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
```